### PR TITLE
Leftover cleanup for .net core 2.1 removals

### PR DESCRIPTION
**Pull request by cijothomas**
Part of https://github.com/open-telemetry/opentelemetry-dotnet/issues/2156

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj [hunk 1]
```diff
@@ -1,26 +1,6 @@
...
-        When the CI runs the integration test, the TARGET_FRAMEWORK variable is set. The presence of this variable is used
-        to determine whether to include the netstandard2.1 target.
-
-        Ideally, this would check the version of the SDK used to build the project rather than a check that is dependent on
-        how the integration test works. I have yet to find an available MSBuild property for checking SDK version.
-    -->
-    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) == ''">$(TargetFrameworks);netstandard2.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) != '' AND $(TARGET_FRAMEWORK) != 'netcoreapp2.1'">netstandard2.1</TargetFrameworks>
-
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj:3]: given .net core 2.1 is no longer required, i wonder if we need to retain the netstandard2.0 at all? As .NET Core 3.1 and above is covered by netstandard2.1, and all .NET Framework is covered by net461

**Review by reyang — APPROVED** [review]

**Comment by reyang** [comment]
Is it possible to link all the related PRs to #2156? Currently it seems most of the clean up PRs are not connected/linked.

**Comment by cijothomas** [comment]
> Is it possible to link all the related PRs to #2156? Currently it seems most of the clean up PRs are not connected/linked.

All 2156 PRs are linked to it. (There is only 2 so far. More might be coming)

https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138 has much more linked PRs, and is all linked to the parent github issue.

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj:3]: keeping netstandard2.0 to help any other libraries which target netstandard2.0 to take a dependency on this.

**Pull request merged**